src/Access/Authentication.cpp

```diff
@@ -65,6 +127,22 @@ namespace
     {
         return checkPasswordDoubleSHA1MySQL(scramble, scrambled_password, Util::encodeDoubleSHA1(password_plaintext));
     }
+
+    bool checkOneTimePassword(const std::string_view & password, const String & secret)
+    {
+        uint8_t buffer[20];
+        uint8_t time_point_bytes[8];
+        auto time_point = getLastTimePoint();
```

Comment:
Just to be explicit and know size, because we operate with bytes from this integer below
```suggestion
        UInt64 time_point = getLastTimePoint();
```